CodeStarterCamp_Week3/CafePosMachine.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  CoffeePosMachine.swift
+//  CodeStarterCamp_Week3
+//
+//  Created by minsong kim on 2023/03/06.
+//
+
+import Foundation
+
+
+struct Person {
+    var money: Int
+    var nickname: String
+    
+    mutating func orderCofffee(menu: String, price: Int) {
+        if price == 0 {
+            print("\(nickname)님, 주문하신 \(menu)는 품절이라 구매하실 수 없습니다. 남은 잔액은 \(money)원 입니다.")
+        }else if money > price {
```

Comment:
중괄호 다음은 띄어쓰는것이 일반적인 컨벤션입니다 ㅎㅎ
```suggestion
        } else if money > price {
```